[{"auth":"Authentication verification ","passed":"false"},{"AssertFail":{"generatedMessage":false,"name":"AssertionError [ERR_ASSERTION]","code":"ERR_ASSERTION","actual":401,"expected":"401","operator":"!="},"message":"Authentication verification ","passed":"false"}]